libraries/lib-wave-track/WaveTrack.h

```diff
@@ -296,12 +296,9 @@ class WAVE_TRACK_API WaveTrack final : public WritableSampleTrack
    // and alignment for efficiency
    //
 
-   sampleCount GetBlockStart(sampleCount t) const override;
-
    // These return a nonnegative number of samples meant to size a memory buffer
-   size_t GetBestBlockSize(sampleCount t) const override;
    size_t GetMaxBlockSize() const override;
-   size_t GetIdealBlockSize();
+   size_t GetMaxBlockSize();
```

Comment:
Why are there now const and non-const overrides?  This suggests the query might mutate the track.

I think `GetIdealBlockSize` was erroneously non-const, and you just made a global search and replace?